descriptors/ModuleDescriptor-template.json

```diff
@@ -80,7 +81,8 @@
             "circulation-storage.loan-policies.item.get",
             "circulation-storage.request-policies.item.get",
             "circulation-storage.fixed-due-date-schedules.item.get",
-            "circulation-storage.fixed-due-date-schedules.collection.get"
+            "circulation-storage.fixed-due-date-schedules.collection.get",
+            "patron-notice.post"
```

Comment:
Please add a dependency for this interface as well

Reply:
Notices are send without waiting for response.
Do you think mod-circulation now has hard dependency on mod-notify?

Reply:
@dmytrotur As far as I am aware, FOLIO only has explicit or implicit dependencies.

Until we have a way of describing optional dependencies, I advocate for explicit dependencies, even if that extends the dependency graph and impacts environment building.

Otherwise it is not possible to tell if the dependency is optional or missing because of a mistake.

Does that make sense?

Are you proposing having an implicit dependency?

Reply:
Yes, make sense for me. I'll add explicit dependency.